spec.md

```diff
@@ -1616,23 +1617,34 @@ A register is initialized with an indeterminate value (see [@sec:indeterminate-v
 
 # Conditionals
 
-Conditional statements define one or more regions where the execution of operations (declarations, statements, and expressions) inside the regions is augmented by a *condition*.
-The evaluation behavior of operations is operation-dependent:
+Conditional statements define one or more regions where the operations (declarations, statements, and expressions) inside a region are *conditionally executed*.
```

Comment:
Bikeshed, but I'm still firmly in the camp that what we're doing is better named "simulation" and not "execution."

Concurrency notwithstanding, "execution" rings to my ears as something inherently sequential. (Maybe it's just because the phrase "to execute a program" is burned into my brain).

Execution may play a part in the simulation (eg, a multi-step task is scheduled, like a procedural block in Verilog). But if I take a circuit diagram, plugged a number on each of its inputs, and computed the resulting outputs, I don't think of that as "executing the circuit."

Reply:
I don't know. The reason why I've been partial to "execute" is that "simulate" seems to narrowly be limited to "simulation" (which seems to pedantically not be what happens if you go and put this FIRRTL on an FPGA). Another alternative could be "evaluate".

That said, this matches the existing language of "execute" which is already in the layers and command sections. Is it fine to punt on this to fix in a follow-on PR on-mass if we decide to change?